### DISCUSSION TITLE
Wall Implementation and representation

### DISCUSSION THREAD

**Discussion by PestyVesty** [mesa/mesa#2556] [General]
Is there a clean way to implement walls in the current mesa framework? I would need walls in my environment for agents to navigate, and thought it may be best to ask before going down a rabbit hole. At the same time, will there be a way to visualize the wall in the environment with the current code base for mesa?

**Comment by EwoutH** [comment]
This is a great question! Just two weeks ago we were discussing this:
- https://github.com/projectmesa/mesa/issues/2534

I will dive into it a bit later today to see what's currently possible.

**Reply by PestyVesty** [reply]
I forgot to mention I was referring to the continuous environment, which may or may not make it easier to implement when compared to a grid environment.

I'll do some research before giving it at implementing walls in continuous environments.
